libs/payment/api/src/lib/payment-provider/bexio/bexio-payment-provider.ts

```diff
@@ -317,12 +334,17 @@ export class BexioPaymentProvider extends BasePaymentProvider {
     addToStringReplaceMap(stringReplaceMap, 'subscription', invoice.subscription)
     addToStringReplaceMap(stringReplaceMap, 'user', invoice.subscription.user)
     addToStringReplaceMap(stringReplaceMap, 'memberPlan', invoice.subscription.memberPlan)
+
     const bexioInvoice: InvoicesStatic.InvoiceCreate = {
       title: isRenewal ? this.invoiceTitleRenewalMembership : this.invoiceTitleNewMembership,
       contact_id: contact.id,
       user_id: this.userId,
       mwst_type: 0,
       mwst_is_net: false,
+      // Override for wrong library implementation!
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
```

Comment:
```suggestion
      // @ts-expect-error
```

Reply:
Will remove that as soon as the fix is merged: https://github.com/mathewmeconry/bexio/pull/61